dashboard/src/components/AboutDialog.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+import { api, client } from "@/client";
+import useEventListener from "@/composables/useEventListener";
+import Modal from "bootstrap/js/dist/modal";
+import { ref, onMounted } from "vue";
+import { closeDialog } from "vue3-promise-dialog";
+
+const el = ref<HTMLElement | null>(null);
+const modal = ref<Modal | null>(null);
+const about = ref<api.EnduroAbout | null>(null);
+
+onMounted(async () => {
+  if (!el.value) return;
+  modal.value = new Modal(el.value);
+  modal.value.show();
```

Comment:
Does this mean the modal is displayed on page load?

Reply:
The component is the actual dialog/modal so it displays on mounted.